handlers/admin_handlers.py

```
@@ -1,178 +1,201 @@
 
-    pending_requests = load_data(PENDING_REQUESTS_FILE)
+    page = context.user_data.get('page', 1)
+    page_size = 10
+    pending_requests, total_count = user_service.get_pending_users(
+        page, page_size)
+
     if not pending_requests:
         text = "Нет ожидающих заявок."
         if callback_query:
-            await callback_query.message.edit_text(text)
+            await callback_query.message.edit_text(text, reply_markup=keyboard)
```

Ruff (0.8.2)

Undefined name `keyboard`

(F821)